client/src/styles/colors.js

```diff
@@ -0,0 +1,17 @@
+export const editor = {
+  background: '#1e2127',
+  currentLineBackground: '#1b1c22',
+  fontColor: '#9ba1a2',
+};
+
+export const sidebar = {
+  background: '#272727',
+  icon: '#878787',
+  iconActive: '#FFFFFF',
+  iconDisabled: '#878787',
+};
+
+export default {
+  editor,
+  sidebar,
+};
```

Comment:
Are these some of the global variables you mentioned?

If we're using Stylus, we can make use of [`.define`](http://stylus-lang.com/docs/js.html#definename-node) for global variables.